modules/tinymce/src/core/main/ts/api/EventTypes.ts

```diff
@@ -23,6 +24,8 @@ export interface NewBlockEvent { newBlock: Element }
 
 export interface NodeChangeEvent { element: Element; parents: Node[]; selectionChange?: boolean; initial?: boolean }
 
+export interface FormatEvent { format: string; vars: FormatVars }
```

Comment:
As discussed the other day, this should also be including the optional node/range passed to the apply/remove functions. Additionally, this type is incorrect as `vars` is optional.

- [ ] Fix to include the optional `node` as discussed
- [ ] Fix incorrect types